src/systemcmds/hardfault_log/CMakeLists.txt

```diff
@@ -33,7 +33,7 @@
 px4_add_module(
 	MODULE systemcmds__hardfault_log
 	MAIN hardfault_log
-	STACK_MAIN 1024
+	STACK_MAIN 2048
```

Comment:
I needed to increase it (crashed before), and adding 1kb is for sure enough for my changes, but I did not check how much is actually needed. My thinking was that it's a short-running command early at boot, so there's for sure enough RAM available. And it's an important task that must not run out of stack ever.
If you want I can check how much is needed exactly.